packages/mdc-toolbar/mdc-toolbar.scss

```diff
@@ -168,10 +168,9 @@
   // bem linter is disabled because it cannot handle multiple top level selector
   &-default-behavior {
     .mdc-toolbar__title {
-      @include mdc-typography(display1);
+      @include mdc-typography(headline6);
```

Comment:
I'm a bit confused why mdc-toolbar__title used to have 2 different typography styles but now doesn't. Was that a bug or intentional, and if it's the same now, then do we even need this mixin here?